packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -59,6 +59,23 @@ describe('Dropdown', () => {
       )
     })
 
+    it('is "false" when closed by toggle indicator click', () => {
+      const wrapper = mountWithProvider(<Dropdown onOpenChange={onOpenChange} items={items} />)
+      const triggerButton = wrapper.find(`button.${Dropdown.slotClassNames.triggerButton}`)
```

Comment:
is it necessary to specify exact DOM elements in the selector? This makes these tests being brittle - for example, if `toggleIndicator` will change its element type from `span` to, say, `i`, it will result in failed test while, in fact, nothing will be broken for this scenario

Reply:
From what I remember, it was, as otherwise it would also find the actual React component, not only my DOM element. I can double check, but I remember adding element type specifically for this reason.

Reply:
<img width="601" alt="Capture" src="https://user-images.githubusercontent.com/11275392/56962583-3b233280-6b57-11e9-8003-3a3592e3298e.PNG">

Reply:
you can use `getDOMNode` for that: https://airbnb.io/enzyme/docs/api/ReactWrapper/getDOMNode.html